pkg/status/dist/templates/header.tmpl

```diff
@@ -55,4 +60,40 @@ NOTE: Changes made to this template should be reflected on the following templat
   {{- if .hostnameStats.errors.all }}
     error: {{.hostnameStats.errors.all}}
   {{- end }}
+
+{{- if .leaderelection}}
+  Leader Election
+  ===============
+    Leader Election Status:  {{.leaderelection.status}}
+    {{- if eq .leaderelection.status "Failing"}}
+    Error: {{.leaderelection.error}}
+    {{else}}
+    Leader Name is: {{.leaderelection.leaderName}}
+    Last Acquisition of the lease: {{.leaderelection.acquiredTime}}
+    Renewed leadership: {{.leaderelection.renewedTime}}
+    Number of leader transitions: {{.leaderelection.transitions}}
+    {{- end}}
+{{- end}}
+
+{{- if .custommetrics }}
```

Comment:
Let's hold off on this. 
I'd like to discuss what can show up in the output of the status command of the agent vs the cluster agent.
Typically, the custom metrics part has nothing to do in the agent's status.

Reply:
"custommetrics" attribute is added to the payload only in the `GetDCAStatus` function here: https://github.com/DataDog/datadog-agent/blob/717c8bd4c94e9dd6c5145e4d52f78c7ce86debbe/pkg/status/status.go#L117. This attribute will never show up in the regular agent status: https://github.com/DataDog/datadog-agent/blob/717c8bd4c94e9dd6c5145e4d52f78c7ce86debbe/pkg/status/status.go#L31